scripts/dev_setup.py

```diff
@@ -25,13 +25,15 @@ def exec_command(command):
 print('Running dev setup...')
 print('Root directory \'{}\'\n'.format(root_dir))
 
+# install general requirements
+exec_command('pip install -r requirements.txt')
```

Comment:
@derekbekoe it was required because otherwise it installs the newest setuptools, screws everything up, and then installs the desired setuptools.  However, it seems that KeyVault no longer breaks even with this change.